InstanceManager.js

```diff
@@ -6,7 +6,19 @@ const LocalRunner = require('./LocalRunner.js');
 const DockerRunner = require('./DockerRunner.js');
 const common = require('./common.js');
 const endpointToUrl = common.endpointToUrl;
+const ERR_IN_SHUTDOWN = 30;
```

Comment:
Is this an error code? May want to add a comment to clarify that. Looks like a magic number otherwise.